jar-filter/build.gradle

```diff
@@ -13,8 +13,9 @@ repositories {
 }
 
 ext {
-    kotlin_metadata_version = '0.1.0'
-    test_kotlin_api_version = '1.3'
+    kotlin_metadata_version = '0.2.0'
+    test_kotlin_api_version = '1.4'
+    test_kotlin_version = '1.4.32'
```

Comment:
Why there is a need for `test_kotlin_version` and why is different from `kotlin_version` that is defined in `gradle.properties`?

Reply:
The `kotlin_version` property is the version of Kotlin that the plugin _itself_ is built with. This must be compatible with the versions of Gradle that the plugin is expected to be used by. For example, Gradle 6.8.3 uses the Kotlin 1.4.20 runtime library and its Kotlin DSL uses the Kotlin 1.3 API.

In contrast, `test_kotlin_version` is the version of Kotlin that I compile all of the test classes with. These are the classes that the `jar-filter` plugin will transform, and are expected to be Kotlin 1.4. In fact, the latest version of the `kotlinx-metadata-jvm` library only supports Kotlin 1.4+ metadata, which means that Kotlin < 1.4 will need to use the plugin from the `release/5.0` branch.